src/coord/localij.rs

```diff
@@ -82,7 +82,7 @@ impl TryFrom<LocalIJK> for CellIndex {
             base_cell.map(BaseCell::is_pentagon).unwrap_or_default();
 
         if dir != Direction::Center {
-            // If the index is in a warped direction, we need to unwarp the base
+            // If the index is in a warped direction, we need to unwrap the base
```

Comment:
this one is actually correct: https://en.wiktionary.org/wiki/unwarp

Reply:
Thanks for catching that!